client/src/App.js

```diff
@@ -26,85 +28,80 @@ class App extends Component {
       <div>
         <Navbar fluid>
           <Navbar.Header>
-            <Navbar.Brand>
-              <a href="#">BitsPlease</a>
+            <Navbar.Brand href='home' className='logo-text'>
```

Comment:
I haven't worked with React-Bootstrap as much and I attempted to add a link to this, to send it home, but I must be doing something incorrectly. It's late and it's probably just a dumb mistake, but if you have some feedback from the experience you've had with it, that would be helpful, so I can fix it.